specification/dimensions/usageunit.md

```diff
@@ -0,0 +1,93 @@
+# Usage Unit
+
+Usage Unit is the unit of measure for the usage or consumption of a resource or service. Usage Unit is used in conjunction with Quantity In Usage Unit to gain understanding of usage or consumption of a given product or service in an environment.
+
+The UsageUnit column MUST be present in the billing data. This column MUST be of type String and MUST NOT contain null values when the ChargeType is 'Usage'. UsageUnit should be expressed as a single unit of measure.  Units of measure used in UsageUnit SHOULD adhere to the values and format requirements specified in the Recommended Values section below.
+
+The Usage Unit for a particular line item may differ from PricingUnit when providers use different units, unit increments, or columns to determine cost. The values in UsageUnit and QuantityInUsageUnit are often listed at a finer granularity or over a different time interval than the PricingUnit and QuantityInPricingUnit. While UsageUnit may appear to relate to pricing and cost, UsageUnit is focused on resource or service consumption.  It is essential not to confuse UsageUnit with PricingUnit which is the basis for determining cost. The UsageUnit column MUST not be used as the basis for determining values related to any pricing or cost metric.
+
+## Column ID
+
+UsageUnit
+
+## Display name
+
+Usage Unit
+
+## Description
+
+The unit of measure for the usage or consumption of a resource or service
+
+## Content constraints
+
+|    Constraint   |      Value      |
+|:----------------|:----------------|
+| Column required | True            |
+| Data type       | String          |
+| Allows nulls    | True            |
+| Value format    | \<not specified> |
+
+## Recommended Values
+
+Usage units SHOULD be expressed as a single unit of measure adhering to one of the following three formats.
+
+* `<plural-units>` - "GB", "Seconds"
+* `<singular-unit> <plural-time-units>` - "GB Hours", "MB Days"
```

Comment:
AWS and GCP use `<singular-unit>-<plural-time-units>` (with a dash separator).  Are we deviating for a reason?

AWS: https://repost.aws/knowledge-center/ebs-volume-charges
GCP: https://cloud.google.com/storage/pricing#price-tables

Reply:
```suggestion
* `<singular-unit>-<plural-time-units>` - "GB Hours", "MB Days"
```

Discussed on VS3 call and aligned on use of dash.